pkg/util/oc/oc_struct.go

```diff
@@ -71,20 +72,51 @@ func (o OC) ApplyTemplateFile(t test.TestHelper, ns string, tmplFile string, inp
 	})
 }
 
+// This function can be called by both ApplyString and ApplyFile. Accept as argument: multiple yaml string or yaml file to apply
+func (o OC) applyWithRetry(t test.TestHelper, ns string, inputKind string, arg string) {
+	t.T().Helper()
+	maxAttempts := 5
+	var attemptT *test.RetryTestHelper
+	warning := false
+	for i := 0; i < maxAttempts; i++ {
+		attemptT = retry.Attempt(t, func(t test.TestHelper) {
+			t.T().Helper()
+			o.withKubeconfig(t, func() {
+				t.T().Helper()
+				if inputKind == "string" {
+					shell.ExecuteWithInput(t, fmt.Sprintf("oc %s apply -f -", nsFlag(ns)), arg)
```

Comment:
Does this command work? Don't you have to wrap `arg` in `<<EOF`?
```suggestion
					shell.ExecuteWithInput(t, fmt.Sprintf(`oc %s apply -f - <<EOF
%s
EOF`, nsFlag(ns)), arg)
```
I just ask, I didn't test this function.

Reply:
It's not needed, I tested and works without the EOF. In fact was working before